src/main/target/ALIENWHOOP/config.c

```diff
@@ -120,7 +120,7 @@ void targetConfiguration(void)
 
         /* Setpoints */
         pidProfile->dtermSetpointWeight = 100;
-        pidProfile->setpointRelaxRatio = 100; // default to snappy for racers
+        pidProfile->setpointRelaxRatio = 100; // default to wonky in middle for freestylers use 0 for racers 
```

Comment:
The change to the comment is fairly redundant

Reply:
Well actually the comment is incorrect... 100 is not 'snappy for racers', it's the opposite.  Must go back to some confusion about what this does when accidentally it was keyed in the wrong way around.  Transition of 100 is the least snappy in the middle because the full damping effect of D is applied to all stick inputs.  It's transition of 0 that results in the most tight / snappy feel in the middle (assuming some non-zero value of weight.

Reply:
Sounds good as well.. maybe just worth removing then. I assume the target maintainer chose this value for a particular reason.